data/free worlds start.txt

```diff
@@ -1615,6 +1821,116 @@ mission "FW Pirates 4.1"
+	source "Longjump"
+	destination "Bourne"
+	deadline 14
+	to offer
+		has "fw pirates: crushed"
+	on fail
+		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	
+	on offer
+		conversation
+			`JJ and Freya meet you on your ship soon after you land on <origin>, accompanied by two uniformed military policemen. 
+			`	"We heard about the hornet's nest you stirred up from the other captains, before we arrested them," says JJ. "Congratulations. you escaped from the disaster brought on by your stupidity, and the pirates are united against us. Now we have to fight a war on two fronts while you sit in prison with Tomek." He gestures to two military policemen, who pull out handcuffs and move toward you.`
+
+			choice
+				`	"What pirates? I destroyed their fleet, freed their slaves, and wiped out their support bases." (show your proof)`
+					goto recording
+				`	(Accept the arrest and go quietly.)`
+					goto arrested
+				`	"You're arresting me? What's the charge?" (try to argue your case)`
+					goto argue
+			label recording
+			`	"This is no time for jokes, <first>," Freya says quietly. "We saw the sensor logs from the surviving ships that got away. You're in enough trouble as it is. Don't make it worse with an obvious lie.`
+			`	"You might want to get the rest of the story before you arrest me," you say, holding up a finger. The military policemen pause and look to Tomek. Before he can wave them on, you call up a recording of the battle. The explosions, the shudder of weapon impacts, and the cries of the dying almost make you think you're back in it, but you manage to shake off the flash-backs. When the final explosion ends, JJ and Freya stare at you, pale and wide-eyed. `
+			`Freya breaks the silence. "I... I don't know what to say. You destroyed the core of the combined pirate fleet?! How? What kind of technology are you packing in this ship?"` 
+			`JJ clears his throat and gathers himself up. "We'll find out soon enough. You're under arrest, Captain <lastname>, for crimes against the Free Worlds Senate. Your ship, and all these captured freighters are impounded. You-" 
+			`"-JJ," you interrupt, "perhaps you should explain this to the freed slaves crewing those captured freighters. They're waiting outside."`
+				goto slaves
+			label arrested
+			`	JJ and his policemen walk you out quietly to a small cabin in your ship. His men strip you and your cabin of weapons and personal effects and lock you in, shutting off all communications and lighting to the room. You've become a prisoner in your own ship, with only a sliver of light coming in from a tiny gap under the door. You eat a meager, bland meal off a tray of food gets pushed under the door an hour later, and you go to sleep wondering what your future will be. `
```

Comment:
Yeah, my first version had the player get thrown in a prison cell. I've revised it to have the player go to sleep hungry instead.